app/jobs/mail_alias_sync_job.rb

```diff
@@ -0,0 +1,41 @@
+class MailAliasSyncJob < ApplicationJob
+  queue_as :default
+
+  def perform(mail_alias_id = nil)
```

Comment:
Shouldn't the parameter be mail_alias_ids (plural)? Since a list can be passed. It is a bit confusing now that it looks like a singular id is passed into the `where` and that you get multiple aliases from that.